test/JDBC/expected/BABEL-4217-vu-verify.out

```diff
@@ -0,0 +1,240 @@
+SELECT * FROM BABEL_4217_vu_prepare_t1;
+GO
+~~START~~
+int#!#varchar
+1#!#John
+2#!#Jane
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t2;
+GO
+~~START~~
+int#!#varchar
+1#!#John
+2#!#Jane
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t3;
+GO
+~~START~~
+int#!#varchar
+1#!#John
+2#!#Jane
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t4;
+GO
+~~START~~
+varchar
+John
+Jane
+~~END~~
+
+ALTER TABLE BABEL_4217_vu_prepare_t4 add id INT PRIMARY KEY IDENTITY(1,1) NOT NULL;
+GO
+SELECT * FROM BABEL_4217_vu_prepare_t4;
+GO
+~~START~~
+varchar#!#int
+John#!#1
+Jane#!#2
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t5;
+GO
+~~START~~
+varchar
+John
+Jane
+~~END~~
+
+ALTER TABLE BABEL_4217_vu_prepare_t5 add id INT PRIMARY KEY NOT NULL IDENTITY(1,1);
+GO
+SELECT * FROM BABEL_4217_vu_prepare_t5;
+GO
+~~START~~
+varchar#!#int
+John#!#1
+Jane#!#2
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t6;
+GO
+~~START~~
+varchar
+John
+Jane
+~~END~~
+
+ALTER TABLE BABEL_4217_vu_prepare_t6 add id INT NOT NULL PRIMARY KEY IDENTITY(1,1);
+GO
+SELECT * FROM BABEL_4217_vu_prepare_t6;
+GO
+~~START~~
+varchar#!#int
+John#!#1
+Jane#!#2
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t7;
+GO
+~~START~~
+int#!#varchar
+1#!#John
+2#!#Jane
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t8;
+GO
+~~START~~
+int#!#varchar
+1#!#John
+2#!#Jane
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t9;
+GO
+~~START~~
+int#!#varchar
+1#!#John
+2#!#Jane
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t10;
+GO
+~~START~~
+varchar
+John
+Jane
+~~END~~
+
+ALTER TABLE BABEL_4217_vu_prepare_t10 add id INT UNIQUE IDENTITY(1,1) NOT NULL;
+GO
+SELECT * FROM BABEL_4217_vu_prepare_t10;
+GO
+~~START~~
+varchar#!#int
+John#!#1
+Jane#!#2
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t11;
+GO
+~~START~~
+varchar
+John
+Jane
+~~END~~
+
+ALTER TABLE BABEL_4217_vu_prepare_t11 add id INT UNIQUE NOT NULL IDENTITY(1,1);
+GO
+SELECT * FROM BABEL_4217_vu_prepare_t11;
+GO
+~~START~~
+varchar#!#int
+John#!#1
+Jane#!#2
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t12;
+GO
+~~START~~
+varchar
+John
+Jane
+~~END~~
+
+ALTER TABLE BABEL_4217_vu_prepare_t12 add id INT NOT NULL UNIQUE IDENTITY(1,1);
+GO
+SELECT * FROM BABEL_4217_vu_prepare_t12;
+GO
+~~START~~
+varchar#!#int
+John#!#1
+Jane#!#2
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t13;
+GO
+~~START~~
+int#!#varchar
+1#!#John
+2#!#Jane
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t15;
+GO
+~~START~~
+varchar
+John
+Jane
+~~END~~
+
+
+ALTER TABLE BABEL_4217_vu_prepare_t15 add id INT PRIMARY KEY NOT NULL CHECK(id < 5) IDENTITY(1,1);
+GO
+SELECT * FROM BABEL_4217_vu_prepare_t15;
+GO
+~~START~~
+varchar#!#int
+John#!#1
+Jane#!#2
+~~END~~
+
+
+SELECT * FROM BABEL_4217_vu_prepare_t16;
+GO
+~~START~~
+varchar
+John
+Jane
+~~END~~
+
+
+-- It should give error (default and idenity cannot be assigned to same column)
+ALTER TABLE BABEL_4217_vu_prepare_t16 add id INT PRIMARY KEY NOT NULL DEFAULT 0 IDENTITY(1,1);
```

Comment:
This is one of the example where we can reduce the number of tables as the ALTER can be tested on the same table we are testing other things such as other ALTER statements, INSERT BULK etc. Let's try to be strict on number of objects we are creating overall.

Reply:
Updated